docs/TLint web/README.adoc

```diff
@@ -0,0 +1,30 @@
+= TLint web
+
+TLint web runs `tlint` checks on a web application running `tlint`
+as web assembly code.
+
+
+== Development
+
+Use `rustup` to install and configure `rust`. Compiled version of
+`rust` distributed in SUSE repositories doesn't match with downloaded
+version of `wasm`.
+
+Get `wasm-pack` link:https://drager.github.io/wasm-pack/[here].
```

Comment:
Updated the link as this one was outdated https://rustwasm.github.io/wasm-pack/installer/